backends/bmv2/run-bmv2-test.py

```diff
@@ -276,6 +294,22 @@ def main(argv):
             else:
                 options.compilerOptions += argv[1].split();
                 argv = argv[1:]
+        elif argv[0] == "--sa":
+            if len(argv) == 0:
+                reportError("Missing argument for --sa option")
+                usage(options)
+                sys.exit(FAILURE)
+            else:
+                options.switchOptions += argv[1].split();
+                argv = argv[1:]
+        elif argv[0] == "--stsa":
```

Comment:
The `--stsa` acronym is difficult to understand, can you spell it out please?

Reply:
Done